query/concurrent_unordered_map.hpp

```diff
@@ -0,0 +1,164 @@
+//  Copyright (c) 2017-2018 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef QUERY_CONCURRENT_UNORDERED_MAP_HPP_
+#define QUERY_CONCURRENT_UNORDERED_MAP_HPP_
+
+#if defined(SUPPORT_HASH_REDUCTION) && defined(RUN_ON_DEVICE)
+#include <hash/concurrent_unordered_map.cuh>
+#include <hash/hash_functions.cuh>
+#endif
+#include <thrust/pair.h>
+#if defined(SUPPORT_HASH_REDUCTION) && defined(RUN_ON_DEVICE)
```

Comment:
merge with line 18?

Reply:
lint requires those header files needs to be in order, cuh is considered as C header file while hpp are considered cpp header file